data_steward/utils/pipeline_logging.py

```diff
@@ -111,3 +159,86 @@ def setup_logger(log_filepath_list, console_logging=True):
     for filename in log_path:
         log_list.append(create_logger(filename, console_logging))
     return log_list
+
+
+def _get_date_str():
+    """
+    Get current date formatted using FILENAME_FMT
+    :return: 
+    """
+    return datetime.today().strftime(FILENAME_FMT)
+
+
+def _get_log_file_path():
+    """
+    Get the abs path of the log file to use. 
+    
+    The location is DEFAULT_LOG_DIR and the file name is the current
+    date formatted using FILENAME_FMT.
+
+    :return: absolute path to the log file
+    """
+    date_str = _get_date_str()
+    return os.path.join(DEFAULT_LOG_DIR, f'{date_str}.log')
+
+
+def _get_config(filename, level):
+    """
+    Get a dictionary which describes the logging configuration
+    
+    :param filename: Create the FileHandler using the specified filename.
+    :param level: Set the root logger level to the specified level 
+                  (i.e. logging.{DEBUG,INFO,WARNING,ERROR}).
+    :return: the default configuration dict
+    """
+    # copy _DEFAULT_CONFIG to avoid modifying it
+    default_config = dict(_DEFAULT_CONFIG)
+    default_config['root']['level'] = level
+    file_log_handler_dict = default_config['handlers'][_FILE_HANDLER]
+    file_log_handler_dict['filename'] = filename
```

Comment:
To be safe and to further establish a convention, I would change these lines to:

```suggestion
    file_log_handler_dict = default_config.get('handlers', {}).get(_FILE_HANDLER, {})
    file_log_handler_dict['filename'] = _get_log_file_path()
```

Reply:
Should we get rid of `_DEFAULT_CONFIG` and instead construct the dict config using procedural logic then? If in the suggestion above `file_log_handler_dict` is empty it will need more attributes added (i.e. `class`, `mode`, `formatter`).